server/blob/blob-objectstorage/src/test/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAOTest.java

```diff
@@ -104,5 +108,23 @@ void failsWithRuntimeExceptionOnCreateContainerTwice() throws Exception {
             .hasCauseInstanceOf(ObjectStoreException.class)
             .hasMessageContaining("Unable to create container");
     }
+
+    @Test
+    void supportsEncryptionWithCustomPayloadCodec() throws Exception {
+        ObjectStorageBlobsDAO encryptedDao = ObjectStorageBlobsDAO
+            .builder(testConfig)
+            .container(containerName)
+            .blobIdFactory(blobIdFactory())
+            .payloadCodec(new AESPayloadCodec(new CryptoConfig(
+                "c603a7327ee3dcbc031d8d34b1096c605feca5e1", "foobar")))
```

Comment:
What are these parameters?

Can you extract it to some constants so that I can easily what it corresponds to?

Maybe also having a build of `CryptoConfig` could help (multiple string parameters)